src/samples/aspnet/Elsa.Samples.AspNet.Tenants.External/Program.cs

```diff
@@ -0,0 +1,143 @@
+using System.Security.Claims;
+using Elsa;
+using Elsa.EntityFrameworkCore.Common;
+using Elsa.EntityFrameworkCore.Extensions;
+using Elsa.EntityFrameworkCore.Modules.Management;
+using Elsa.EntityFrameworkCore.Modules.Runtime;
+using Elsa.Extensions;
+using Elsa.Tenants.Extensions;
+using Elsa.Tenants.Helpers;
+using FastEndpoints.Swagger;
+using Microsoft.AspNetCore.Authentication.JwtBearer;
+using NSwag;
+using NSwag.AspNetCore;
+using NSwag.Generation.Processors.Security;
+
+WebApplicationBuilder builder = WebApplication.CreateBuilder(args);
+ConfigurationManager configuration = builder.Configuration;
+
+var identitySection = configuration.GetSection("Identity");
+
+// Add services to the container.
+builder.Services.AddControllers();
+builder.Services.AddElsa(elsa =>
+{
+    ElsaDbContextOptions dbContextOptions = EfCoreDbContextBuilder.BuildDbContextTenantOption();
+    string postgresConnectionString = configuration.GetConnectionString("Postgres")!;
+    string schema = configuration.GetConnectionString("Schema")!;
+
+    if (!string.IsNullOrEmpty(schema))
+    {
+        dbContextOptions.SchemaName = schema;
+        dbContextOptions.MigrationsAssemblyName = typeof(Program).Assembly.GetName().Name;
+    }
+
+    elsa.UseWorkflowManagement(management => management.UseEntityFrameworkCore(ef => ef.UsePostgreSql(postgresConnectionString, dbContextOptions)));
+    elsa.UseWorkflowRuntime(runtime => runtime.UseEntityFrameworkCore(ef => ef.UsePostgreSql(postgresConnectionString, dbContextOptions)));
+
+    elsa.UseIdentity(options => identitySection.Bind(options));
+
+    elsa.UseTenants(configuration => configuration
+            .UseExternalTenantProvider(options => identitySection.Bind(options))
+            .UseEfcoreStrategies());
```

Comment:
I suggest using the casing `UseEFCoreStrategies` for this identifier. Since "EF" stands for "Entity Framework," and our naming convention typically capitalises each letter in two-letter abbreviations, this format aligns well with our standards. It's consistent with how we handle other abbreviations like "IO" for "Input Output" and "UI" for "User Interface."